src/correlator_bosonic_recursive.jl

```diff
@@ -234,4 +277,4 @@ function add_operator_fermi(
       end
     end
   end
-end
+end
```

Comment:
Looks like something strange happened with the formatting here.

Reply:
Still seems like Github is marking this, is there something strange with the formatting here?